nautobot_chatops_ipfabric/worker.py

```diff
@@ -27,6 +30,12 @@
     verify=settings.PLUGINS_CONFIG["nautobot_chatops_ipfabric"].get("IPFABRIC_VERIFY"),
 )
 
+ipfabric_client = IPFClient(
```

Comment:
add a TODO note to make clear that this is a transition, and eventually the `ipfabric_api` will be removed